script/bootstrap.sh

```diff
@@ -130,15 +129,12 @@ elif [ "$(uname)" = "Darwin" ]; then
         brew unlink cmake
         brew install cmake --HEAD
     }
-
-    ## Install coreutils for realpath
-    brew install coreutils
```

Comment:
Is this intentional? Does "realpath" come with macos 13?

Reply:
yes, it is an intended behavior.  The `coreutils` has been installed in following `brew install xxx` command in line113. we no need to reinstall it again after changing `swig@4` to `swig`.

```
 brew install coreutils \
                 readline \
                 cmake \
                 ninja \
                 swig  \
                 lcov && true
```